### PR TITLE
Feature/chart

### PR DESCRIPTION
**Pull request by Partysun**

### CODE REVIEW

File: lib/sanbase_web/graphql/resolvers/price_resolver.ex [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+
+    result_btc = get_price_points(pair_btc, from, to, interval)
+    result_usd = get_price_points(pair_usd, from, to, interval)
+
+    # The data is always in _USD and _BTC measurements, but with a slightly different
+    # timestamp. Zip combines corresponding results
+    result =
+      Enum.zip(result_btc, result_usd)
+      |> Enum.map(fn {%{datetime: ltime} = btc_map, %{datetime: rtime, price_usd: price_usd}} ->
+           IO.inspect(DateTime.to_unix(ltime) - DateTime.to_unix(rtime))
```
Comment (valo) [inline lib/sanbase_web/graphql/resolvers/price_resolver.ex]: I think this inspect needs to be removed

File: lib/sanbase_web/graphql/resolvers/price_resolver.ex [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+  end
+
+  defp get_price_points(pair, from, to, interval) do
+    Sanbase.Prices.Store.fetch_prices_with_resolution(
+      pair,
+      from,
+      to,
+      interval
+    )
+    |> IO.inspect()
```
Comment (valo) [inline lib/sanbase_web/graphql/resolvers/price_resolver.ex]: I think this inspect needs to be removed

**Comment by IvanIvanoff** [comment]
This PR uses an outdated version of the graphql price API. Lets review the PR for the API and address all issues with it there.

**Comment by valo** [comment]
@IvanIvanoff So we should wait with this PR until #106 is merged?

**Comment by valo** [comment]
@Partysun So the pricing GQL API is merged now. Can we merge this?

**Pull request merged**